site/styles/_carousel.scss

```diff
@@ -96,8 +96,7 @@ s72-carousel {
   @extend .carousel-bottom-gradient;
 
   bottom: 0;
-  display: flex;
-  flex-direction: column;
+  display: grid;
```

Comment:
Does this change follows these principles? https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Relationship_of_Grid_Layout

Reply:
It does if we assume that the 2-dimensional layout is necessary.  I prefer the 2-dimensional layout because it means we don't have to worry about dirty hacks with margins and absolute positions etc.

Reply:
I think its incorrect that we're considering negative margins and absolute positions as dirty hacks. A quick google finds consensus that they're not bad practice and I consider them the most suitable approach in many scenarios.

We should use grid for grid layouts, and reconsider if its good practice otherwise.

Reply:
Yes it's good practice because it has two rows now.  :P